src/main/java/org/broadinstitute/hellbender/utils/GATKProtectedMathUtils.java

```diff
@@ -47,6 +48,16 @@ public static double logSumExp(final Collection<Double> values) {
         return max + Math.log(sum);
     }
 
+    /**
+     * Returns log(max(abs(x), minLogArg))
+     * @param x argument of log (any double value)
+     * @param minLogArg minimum argument to pass to log (must be positive but not checked)
+     * @return any double value.
+     */
+    public static double safeLog(final double x, final double minLogArg) {
```

Comment:
For example it seem that you use the Math.abs to make sure you don't deal with a negative probability but in that case in stead of simply set it two 0, just "bounce" it back to its positive absolute value? 

So if I pass you a "probability" of say -.25 is as good as if a had pass you .25?